### ISSUE TITLE
Compile against latest eduke32 master

### ISSUE THREAD

**Issue by shoober420** [nukeykt/Rednukem#18]
Is it possible that rednukem can pull from the latest eduke32 svn master and compile against it?

How will rednukem be updated when eduke32 keeps releasing later versions?

**Comment by nukeykt** [comment]
I've added rednukem code to nblood repo a while ago, but it does not compile right now due to eduke32 recent changes. You probably could bisect and find working revision

**Comment by shoober420** [comment]
Are you saying that I can play Duke Nukem under NBlood, and that NBlood is the continuation of rednukem?

**Comment by nukeykt** [comment]
No, it means that you can type `make rednukem.exe` in nblood source folder and rednukem exe with more up to date engine revision will be compiled

**Comment by shoober420** [comment]
Is there a way to attempt compilation of NBlood against eduke32 master, even if it fails? I can post here or eduke32 forums with my findings.
